src/dashboard/infrastructure/analytics-4/site-kit-analytics-4-adapter.php

```diff
@@ -76,11 +84,13 @@ public function get_comparison_data( Analytics_4_Parameters $parameters ): Data_
 	public function get_daily_data( Analytics_4_Parameters $parameters ): Data_Container {
 		$api_parameters = $this->build_parameters( $parameters );
 
-		$response = self::$analytics_4_module->get_data( 'report', $api_parameters );
+		$request = new WP_REST_Request( 'GET', self::ANALYTICS_DATA_REPORT_ROUTE );
```

Comment:
Is there a reason we don't apply the same `api-call` abstraction as we do in the Seach Console adapter here?

Reply:
Commenting to the thread because I'm interested in that answer myself 😅